app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  after_create :update_likes_counter
+
```

Comment:
- [OPTIONAL] It would be great to add the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call #update_likes_counter method every time a new like has been saved to the database. To do this you should add a single line of code, public, like this:
`after_save :update_like_counter`

Reply:
Thank you @BregornOriginal for the constructive feedback.I have resolved all the above optional comments.